src/tools/perf/api/libperf.h

```diff
@@ -98,6 +98,29 @@ enum {
 };
 
 
+#define FOREACH_UCP_PERF_DAEMON_AM_ID \
+    /* Host to daemon communication */  \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_INIT,     0) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_REQ,      1) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_SEND_ACK, 2) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_RECV_ACK, 3) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_FIN,      4) \
+\
+    /* Peer to peer daemon communication */ \
+    X(UCP_PERF_DAEMON_AM_ID_PEER_INIT,     5) \
+    X(UCP_PERF_DAEMON_AM_ID_PEER_OP,       6)
+
+
+typedef enum {
+#define X(ID, VALUE) ID = VALUE,
+    FOREACH_UCP_PERF_DAEMON_AM_ID
+#undef X
+} ucp_perf_daemon_am_id_t;
+
+
+const char *ucp_perf_daemon_am_id_name(ucp_perf_daemon_am_id_t id);
```

Comment:
move this prototype below, along with other prototypes

Reply:
ok